testing/disk/default.nix

```diff
@@ -3,24 +3,32 @@
 
 , install-playos
 
-, # Size of data partition in GiB
-  dataPartitionSize ? 5
+, closureInfo
+, systemImage
 
-, # Size of system partition in GiB
-  systemPartitionSize ? 10
+, # Size of data partition in GB
+  dataPartitionSize ? 1
+, bootPartitionSize ? 1
 }:
 with lib;
 let
-  # disk size in GiB
-  diskSize = (1 + dataPartitionSize + systemPartitionSize + systemPartitionSize + 1);
+  systemClosureInfo = closureInfo { rootPaths = [ systemImage ]; };
+  systemSizeBytes = strings.toInt (builtins.readFile "${systemClosureInfo}/total-nar-size");
+  systemSizeGB = systemSizeBytes / (1000.0*1000*1000);
+  # add extra 20% because nar size is not accurate
```

Comment:
Nope, I think GB/GiB has nothing to do with this (although the install script should really be fixed to use GiB's rather than GB's?) and if it did, you would get a bigger `systemSizeGB` anyway (since it is divided by a smaller number). You can compare your own `/nix/store` size with say `du -s -b /nix/store` and `nix path-info --json --all | jq 'map(.narSize) | add'`. For me disk usage is always bigger than what NAR size sum is, although less radically so than I have seen for specific images.

Reply:
OK. Weird!

Reply:
Did some extra checks, `du` reported size differed by 5-8% consistently, so I am keeping this at 20% just to be on the safe side.